app/src/common/shared/com/igalia/wolvic/utils/ConnectivityReceiver.java

```diff
@@ -29,6 +30,11 @@ public ConnectivityReceiver(@NonNull Context context) {
     }
 
     public void init() {
+        // TODO: CONNECTIVITY_ACTION constant was deprecated in API level 28.
+        //  apps should use the more versatile requestNetwork(NetworkRequest, PendingIntent),
+        //  registerNetworkCallback(NetworkRequest, PendingIntent) or
+        //  registerDefaultNetworkCallback(NetworkCallback) functions instead for faster and more
+        //  detailed updates about the network changes they care about.
```

Comment:
Maybe include this in the new issue about connectivity and link it here

Reply:
https://github.com/Igalia/wolvic/issues/803